src/language/JSUtils.js

```diff
@@ -21,15 +21,15 @@
  *
  */
 
-/*jslint regexp: true */
-
 /**
  * Set of utilities for simple parsing of JS text.
  */
 define(function (require, exports, module) {
     "use strict";
 
-    var _ = require("thirdparty/lodash");
+    var _          = require("thirdparty/lodash"),
+        AcornLoose = require("node_modules/acorn/dist/acorn_loose"),
+        ASTWalker  = require("node_modules/acorn/dist/walk");
```

Comment:
I'm quite sure this will not work on dist.
You should add some logic here:
https://github.com/adobe/brackets/blob/abdf55d51f7c4bb6160a4a7b20048d8d0ec31e08/Gruntfile.js#L143

Maybe going forward we should use more webpack, but in another PR.